File: app-java/src/main/java/tech/thdev/java_udemy_sample/view/sample/SampleActivity.java
package tech.thdev.java_udemy_sample.view.sample;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import tech.thdev.java_udemy_sample.R;
import tech.thdev.java_udemy_sample.databinding.ActivitySampleBinding;
import tech.thdev.java_udemy_sample.util.ActivityUtil;

/**
 * Created by tae-hwan on 10/29/16.
 */

public class SampleActivity extends AppCompatActivity {

    private ActivitySampleBinding activitySampleBinding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        activitySampleBinding = ActivitySampleBinding.inflate(getLayoutInflater());
        setContentView(activitySampleBinding.getRoot());

        activitySampleBinding.toolbar.setTitle(R.string.app_name);
        setSupportActionBar(activitySampleBinding.toolbar);

        ActivityUtil.replaceFragmentToActivity(getSupportFragmentManager(), SampleFragment.getInstance(), R.id.frame_layout);
    }
}

File: app-kotlin/src/main/java/tech/thdev/kotlin_udemy_sample/view/sample/SampleActivity.kt
package tech.thdev.kotlin_udemy_sample.view.sample

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import tech.thdev.kotlin_udemy_sample.R
import tech.thdev.kotlin_udemy_sample.databinding.ActivityImageSampleBinding
import tech.thdev.kotlin_udemy_sample.util.replaceFragmentToActivity

class SampleActivity : AppCompatActivity() {

    private lateinit var activityImageSampleBinding: ActivityImageSampleBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        activityImageSampleBinding = ActivityImageSampleBinding.inflate(layoutInflater)
        setContentView(activityImageSampleBinding.root)

        activityImageSampleBinding.toolbar.setTitle(R.string.app_name)
        setSupportActionBar(activityImageSampleBinding.toolbar)

        replaceFragmentToActivity(SampleFragment.getInstance(), R.id.frame_layout)
    }
}
